### PR TITLE
Add guide for VSCode extension

### PR DESCRIPTION
**Pull request by nmattela**

### CODE REVIEW

File: docs/guide/vscode-getting-started.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    - **Windows** users should additionally install [Visual C++ Build Tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019)
+    - **Linux** users should additionally install [x86_64-w64-mingw32-cmake](https://aur.archlinux.org/packages/mingw-w64-cmake)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Sapiens VSCode Extension](https://marketplace.visualstudio.com/items?itemName=Sapiens-OSS.sapiens-vscode-extension)
+
+Ensure that `steam` is a recognized command on your system. If not, edit your PATH system environment variable to point to your Steam binary.
+- On **Windows**, your Steam binary is likely located at `C:\Program Files (x86)`, which is not by default in your PATH.
+
+Please take note of your Sapiens installation's mod directory:
+- For **Windows** users, this is by default installed at `C:\Users\<your_username>\AppData\majicjungle\sapiens\mods`.
```
Comment (suppergerrie2) [inline docs/guide/vscode-getting-started.md]:
```suggestion
- For **Windows** users, this is by default installed at `%AppData%\majicjungle\sapiens\mods`.
```
You can use `%AppData%` to get into the Roaming folder. 
The path posted here is incorrect as it's missing Roaming in the path

**Review by suppergerrie2 — CHANGES_REQUESTED** [review]

File: docs/guide/vscode-getting-started.md (hunk 1 above)
Comment (nmattela) [reply docs/guide/vscode-getting-started.md]: Let me test if using `%AppData%` works with Node.JS's filesystem library first... I'm not sure if it understands that

Comment (nmattela) [reply docs/guide/vscode-getting-started.md]: Ok, that surprisingly enough works. Will add it

**Review by suppergerrie2 — APPROVED** [review]

**Review by DecDuck — APPROVED** [review]
Looks good

**Pull request merged**